minicluster/src/main/java/alluxio/master/LocalAlluxioCluster.java

```diff
@@ -112,8 +112,7 @@ public WorkerNetAddress getWorkerAddress() {
   protected void startMasters() throws Exception {
     mMaster = LocalAlluxioMaster.create(mWorkDirectory);
     mMaster.start();
-    // TODO(peis): Reenable this. This is slowing down the tests.
-    // mMaster.startSecondary();
+    mMaster.startSecondary();
```

Comment:
looking at the code, we are stopping secondary master in the stop method while we have separate start and startSecondary master methods ... any reason for this assymetry? if not, please fix that, thanks